README.md

```diff
@@ -1,6 +1,6 @@
 # Consent Ontology Services
 
-[![Build Status](https://travis-ci.com/broadinstitute/consent-ontology.svg?token=3ve6QNemvC5zpJzsoKzf&branch=develop)](https://travis-ci.com/broadinstitute/consent-ontology)
+[![Build Status](https://travis-ci.org/DataBiosphere/consent-ontology.svg?token=3ve6QNemvC5zpJzsoKzf&branch=develop)](https://travis-ci.org/DataBiosphere/consent-ontology)
```

Comment:
also maybe add the coverage badge? something like:
```
[![Coverage Status](https://coveralls.io/repos/DataBiosphere/consent-ontology/badge.svg?branch=develop)](https://coveralls.io/r/DataBiosphere/consent-ontology?branch=develop)
```

Reply:
Similar to the comment in consent, we can't link there without a new coveralls account being configured for `DataBiosphere`. Currently looking at other ways to accomplish this same task.